File: part 3/practice/Token.cs
using System;


    public class Token
    {
        public static string PLUS = "PLUS";
        public static string MINUS = "MINUS";
        public static string INT = "INT";

        public string Type { get; set; }
        public string Value { get; set; }

        public Token(string type, string value) {
            this.Type = type;
            this.Value = value;
        }

        public override string ToString()
        {
            return $"Type: {Type}, Value: {Value}";
        }
    }

File: part 7 - AST/lesson/Token.py
class Token(object):
    INTEGER, PLUS, MINUS, MUL, DIV, LPAREN, RPAREN, EOF = (
        'INTEGER',
        'PLUS',
        'MINUS',
        'MUL',
        'DIV',
        '(',
        ')',
        'EOF',
    )

    def __init__(self, type, value):
        self.type = type
        self.value = value

    def __str__(self):
        """String representation of the class instance.

        Examples:
            Token(INTEGER, 3)
            Token(PLUS, '+')
            Token(MUL, '*')
        """
        return 'Token({type}, {value})'.format(
            type=self.type,
            value=repr(self.value),
        )

    def __repr__(self):
        return self.__str__()
